server/database/migrations/20190610091811_create_users_cards.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('users_cards', (table) => {
+    table.increments();
+    table
+      .integer('user_id')
+      .notNull()
+      .references('id')
+      .inTable('users');
+    table
+      .integer('card_id')
+      .notNull()
+      .references('id')
+      .inTable('cards');
+    table.boolean('success').notNull();
```

Comment:
Should make these counters so we know total tries & total correct for users_cards
table.integer('successes').notNull();
table.integer('attempts').notNull();

Reply:
oh, looks like this maybe is already there? Confused by how this pull request is sorted...